src/utils/getFillScale.ts

```diff
@@ -25,6 +26,17 @@ export const getFillScale = (
   return getColorScale(Object.keys(domain), colors)
 }
 
+export const getFillScaleForScatter = (
```

Comment:
This is great, I want to use this style for the line layer too. Want to make that change in this PR? If not I can do it separately.

Reply:
I just wanted to you to look at it first, but I can totally make that change in this PR, no problem.